app/src/main/java/org/mozilla/fenix/Config.kt

```diff
@@ -47,10 +45,15 @@ object Config {
         "fenixNightly" -> ReleaseChannel.FenixNightly
         "debug" -> ReleaseChannel.FenixDebug
         "fennecProduction" -> ReleaseChannel.FennecProduction
+
+        // Builds for local performance analysis, recording benchmarks, automation, etc.
+        // This should be treated like a released channel because we want to test
+        // what users experience and there are performance-impacting changes in debug
+        // release channels (e.g. logging) that are never intended to be shipped.
+        "forPerformanceTest" -> ReleaseChannel.FenixProduction
+
```

Comment:
I actually am not sure what to think about this. I am debating whether this should be a separate enum value entirely. 

Obviously the performance test variant should mimic a released variant, but I am not sure that they should be equal. 

What do you think about making the performance variant a separate enum and then modifying the enum class's methods to take that into account when determining the values of isRelease() and isDebug() etc?

Reply:
I originally wrote it that way and decided not to because, if additional methods are added or someone checks `Config.channel` directly, they'd almost certainly not take into account the performance variant.

If we were to make it a separate enum value, I'd want to refactor the code such that `Config.channel` is not exposed so the only entry points are those `isReleaseOrDebug` wrapping functions and the `when` statements are exhaustive so devs will, by following convention, have to consider the `forPerformanceTest` builds.

I think I'd prefer that, tbh, but felt some resistance (from myself) towards doing the refactor – e.g. something I didn't understand or might take too long to do – but I can go back and do it if you'd like.

Reply:
I think that we should make it a separate enum value and do the refactor. However, I guess the expediency/easiness argument will win.

Reply:
I filed https://github.com/mozilla-mobile/fenix/issues/7416 for the follow-up, thanks.